src/Term/AliasGroup.php

```diff
@@ -19,7 +19,7 @@
 class AliasGroup implements Comparable, Countable {
 
 	/**
-	 * @var string
+	 * @var string Requested and actual language of the aliases.
```

Comment:
> you are putting the documentation of what the field means in the subclass here

Yes and no. The sentence here is true with and without the subclass.
> document this in the derivative.

It is, or isn't it?